### PR TITLE
K8S1.9.4 + ETCD 3.2.17 + K8S dashboard 1.8.3

### PR DESCRIPTION
**Pull request by ydye**

### CODE REVIEW

File: kubernetes-deployment/template/kubelet.sh.template [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
 docker run \
     --name=kubelet \
-    --volume=/etc/kubernetes/manifests:/etc/kubernetes/manifests:rw \
```
Comment (fanyangCS) [inline kubernetes-deployment/template/kubelet.sh.template]: >manifests [](start = 29, length = 9)

why do we change it? etcd 3 removes it?

File: kubernetes-deployment/template/kubelet.sh.template [hunk 2]
```diff
@@ -36,7 +36,8 @@ docker run \
     {{ clusterconfig['dockerregistry'] }}/hyperkube:{{ clusterconfig['hyperkubeversion'] }} \
     /hyperkube kubelet --address={{ hostcofig['hostip'] }} \
   --register-node=true \
-  --api-servers={{ clusterconfig['api-servers-ip'] }}:8080 \
+  --kubeconfig=/etc/kubernetes/config \
```
Comment (fanyangCS) [inline kubernetes-deployment/template/kubelet.sh.template]: >kubeconfig=/etc/kubernetes/config \ [](start = 4, length = 35)

if we change to a kubeconfig, should we include an example kube config ?

File: kubernetes-deployment/template/kubelet.sh.template (hunk 1 above)
Comment (ydye) [reply kubernetes-deployment/template/kubelet.sh.template]: To mount the kubeconfig

File: kubernetes-deployment/template/kubelet.sh.template (hunk 2 above)
Comment (ydye) [reply kubernetes-deployment/template/kubelet.sh.template]: K8s1.9 has removed the option api-servers. Instead, we should write the ip-address in the kubeconfig file.

Comment (ydye) [reply kubernetes-deployment/template/kubelet.sh.template]: The same configuration of kubectl

Comment (fanyangCS) [reply kubernetes-deployment/template/kubelet.sh.template]: do we have an example kubeconfig for kubectl?

---
In reply to: [175327008](https://github.com/Microsoft/pai/pull/394#discussion_r175327008) [](ancestors = 175327008)

Comment (ydye) [reply kubernetes-deployment/template/kubelet.sh.template]: Yes

[the remark this answers is not in the document]
Comment (ydye) [reply kubernetes-deployment/template/kubelet.sh.template]: yes

**Pull request merged**